TSOClient/tso.client/UI/Profile/UIJobInfo.cs

```diff
@@ -0,0 +1,377 @@
+                PerformanceTitle.Alignment = TextAlignment.Left | TextAlignment.Middle;
+                PerformanceTitle.Caption = "Performance";
+                this.Add(PerformanceTitle);
+
+                int progressBarWidth = 145;
+                ProgressBar = new UIProgressBar()
+                {
+                    X = 360 - progressBarWidth,
+                    Y = verticalSpace + performanceTitleSectionHeight - 5,
+                    Value = 0
+                };
+                ProgressBar.SetSize(progressBarWidth, 27);
+                ProgressBar.Caption = "";
+                
+                ProgressBar.Background = TextureGenerator.GenerateRoundedRectangle(GameFacade.GraphicsDevice, new Color(37, 58, 77), progressBarWidth, 27, 9);
+                this.Add(ProgressBar);
+
+                verticalSpace += sectionHeight + standardVerticalSpace;
+
+                UIImage NextPositionSectionImage = new UIImage(GetTexture((ulong)0x7A400000001)).With9Slice(13, 13, 13, 13);
+                NextPositionSectionImage.SetSize(360, sectionHeight);
+                NextPositionSectionImage.Position = new Vector2(20, verticalSpace);
+                Add(NextPositionSectionImage);
+                this.Add(NextPositionSectionImage);
+
+                int nextPositionTitleSectionHeight = (sectionHeight / 2) - 8;
+
+                NextPositionTitle = new UILabel
+                {
+                    Position = new Vector2(40, verticalSpace + nextPositionTitleSectionHeight),
+                    Size = new Vector2(200, 12),
+                    CaptionStyle = TextStyle.DefaultTitle
+                };
+                NextPositionTitle.CaptionStyle = NextPositionTitle.CaptionStyle.Clone();
+                NextPositionTitle.CaptionStyle.Color = new Color(238, 247, 169);
+                NextPositionTitle.CaptionStyle.Size = 14;
+                NextPositionTitle.CaptionStyle.Shadow = false;
+                NextPositionTitle.Alignment = TextAlignment.Left | TextAlignment.Middle;
+                NextPositionTitle.Caption = "Next Position";
+                this.Add(NextPositionTitle);
+
+                NextPosition = new UILabel
+                {
+                    Position = new Vector2(40, verticalSpace + nextPositionTitleSectionHeight),
+                    Size = new Vector2(320, 12),
+                    CaptionStyle = TextStyle.DefaultTitle
+                };
+                NextPosition.CaptionStyle = NextPosition.CaptionStyle.Clone();
+                NextPosition.CaptionStyle.Color = new Color(238, 247, 169);
+                NextPosition.CaptionStyle.Size = 12;
+                NextPosition.CaptionStyle.Shadow = false;
+                NextPosition.Alignment = TextAlignment.Right | TextAlignment.Middle;
+                this.Add(NextPosition);
+            }
+        }
+
+        public override void Removed()
+        {
+            base.Removed();
+            Icon.Texture?.Dispose();
```

Comment:
Where is Icon defined? Is it ever set?

Reply:
Removed Icon Dispose; Replaced with Dispose for bespoke texture for progress bar.